### PR TITLE
Учебный проект: нас орда

### PR DESCRIPTION
**Pull request by gromniki**
---
:mortar_board: [Учебный проект: нас орда](https://up.htmlacademy.ru/javascript/17/user/213549/tasks/4)

:boom: https://htmlacademy-javascript.github.io/213549-code-and-magick-17/3/

### CODE REVIEW

File: js/setup.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+var COAT_COLOR = [
+  'rgb(101, 137, 164)',
+  'rgb(241, 43, 107)',
+  'rgb(146, 100, 161)',
+  'rgb(56, 159, 117)',
+  'rgb(215, 210, 55)',
+  'rgb(0, 0, 0)'
+];
+
+var EYES_COLOR = [
```
Comment (rkhaslarov) [inline js/setup.js]: EYES_COLORS

File: js/setup.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  'Верон',
+  'Мирабелла',
+  'Вальц',
+  'Онопко',
+  'Топольницкая',
+  'Нионго',
+  'Ирвинг'
+];
+
+var COAT_COLOR = [
```
Comment (rkhaslarov) [inline js/setup.js]: COAT_COLORS

File: js/setup.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  'yellow',
+  'green'
+];
+
+var getRandomNumber = function (min, max) {
+  return Math.floor(Math.random() * (max - min)) + min;
+};
+
+var getRandomFullName = function (names, surnames) {
+  var name = names[getRandomNumber(0, names.length)];
```
Comment (rkhaslarov) [inline js/setup.js]: can be moved to the separate function return array[getRandomNumber(0, names.length)]

File: js/setup.js [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+  var wizardElement = similarWizardTemplate.cloneNode(true);
+
+  wizardElement.querySelector('.setup-similar-label').textContent = wizard.name;
+  wizardElement.querySelector('.wizard-coat').style.fill = wizard.coatColor;
+  wizardElement.querySelector('.wizard-eyes').style.fill = wizard.eyesColor;
+
+  return wizardElement;
+};
+
+var fragment = document.createDocumentFragment();
```
Comment (rkhaslarov) [inline js/setup.js]: move it to renderWizards function

File: js/setup.js [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+var userDialog = document.querySelector('.setup');
+userDialog.classList.remove('hidden');
+
+var similarListElement = userDialog.querySelector('.setup-similar-list');
+
+var similarWizardTemplate = document.querySelector('#similar-wizard-template')
+  .content
+  .querySelector('.setup-similar-item');
+
+var wizards = [
```
Comment (rkhaslarov) [inline js/setup.js]: can't you just use for loop for that? and please create function generateWizardsData

File: js/setup.js (hunk 1 above)
Comment (gromniki) [reply js/setup.js]: Oops... Rightly

File: js/setup.js (hunk 3 above)
Comment (gromniki) [reply js/setup.js]: What do you mean?

Comment (gromniki) [reply js/setup.js]: corrected

**Pull request merged**